javav2/example_code/dynamodb/checkstyle.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
```

Comment:
Did you mean to put this file here? It seems out of place.

Reply:
One should be in every java project that has a reference for it, and [there are many](https://github.com/awsdocs/aws-doc-sdk-examples/search?p=2&q=checkstyle.xml).

But I removed it, so we can move on so I can get the developer guide updated.